vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/base_rule.rb

```diff
@@ -12,30 +12,28 @@ def initialize(user, card_reader, params = {})
         @user = user
         @card_reader = card_reader
         @params = params
+        @verdict = Verdict.new(@user, @card_reader)
       end
 
       def call
-        evaluate || pass
+        evaluate
```

Comment:
I'm kind of torn, I think I actually like the previous version where verdicts were more immutable, even if they had more constructor arguments. It's not so bad because the verdict only gets mutated in the same class it gets instantiated.

Thoughts?